.gitignore

```diff
@@ -22,3 +22,7 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+.asv
```

Comment:
What generates the `.asv` files?

Reply:
Should not be there as we are not using asv.